inc/item_ticket.class.php

```diff
@@ -1011,25 +1011,30 @@ static function dropdown($options = []) {
 
       $itemtypes = ['Computer', 'Monitor', 'NetworkEquipment', 'Peripheral', 'Phone', 'Printer'];
 
-      $query = "";
+      $union = new \QueryUnion();
```

Comment:
It will fail, see https://github.com/glpi-project/glpi/pull/4934/files#diff-0ddc2f5346f2b34c19111f2775d2330fR55.

I think the constructor should accept empty array now.